src/Tribe/Service_Providers/Context.php

```diff
@@ -241,8 +269,8 @@ static function ( $data ) {
 					'read' => [
 						Tribe__Context::FUNC => static function () {
 							return Dates::build_date_object()
-										->setTime( 0, 0, 0 )
-										->format( Dates::DBDATETIMEFORMAT );
+							            ->setTime( 0, 0, 0 )
+							            ->format( Dates::DBDATETIMEFORMAT );
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 22f2408258849e9f7407ea68c0af5c84</sub>